### PR TITLE
feat: support relative path on windows

### PR DESCRIPTION
**Pull request by fengmk2**

### CODE REVIEW

**Comment by fengmk2** [comment]
https://github.com/grpc/grpc/blob/5facbba8761b7e4f300ff10e61968569f2f0b8bd/package.json#L27

File: package.json [hunk 1]
```diff
@@ -14,11 +14,12 @@
...
   },
   "dependencies": {
-    "is-type-of": "^1.0.0"
+    "debug": "^2.6.8",
+    "is-type-of": "^1.1.0"
   },
   "devDependencies": {
     "autod": "*",
-    "egg-bin": "^2.2.1",
+    "egg-bin": "1",
```
Comment (fengmk2) [inline package.json]: need to support node 4

**Comment by fengmk2** [comment]
1.3.0

**Comment by atian25** [comment]
💯

**Pull request merged**